src/CreeDictionary/CreeDictionary/paradigm/manager.py

```diff
@@ -61,6 +63,29 @@ def sizes_of(self, paradigm_name: str) -> Collection[str]:
         """
         return self._layout_sizes_or_raise(paradigm_name).keys()
 
+    def has_multiple_sizes_for(self, paradigm_name: str) -> bool:
+        """
+        Returns true if the given paradigm comes in more than one size.
+
+        :raises ParadigmDoesNotExistError: when the paradigm name cannot be found.
+        """
+        return len(self.sizes_of(paradigm_name)) > 1
+
+    def default_size_of(self, paradigm_name: str) -> str:
+        """
+        Returns the default size of the paradigm. For paradigms with multiple sizes,
+        this is the first (See ParadigmManagerWithExplicitSizes if you want more
+        control over this).
+
+        :raises ParadigmDoesNotExistError: when the paradigm name cannot be found.
```

Comment:
How do these raise errors without explicitly raising errors? I would like to learn

Reply:
`sizes_of()` may raise this error, so it's possible that this _also_ raises the exact same error.